<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="imageLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QComboBox" name="methodComboBox">
      <item>
       <property name="text">
        <string>Circularity Detection</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hough Method</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="chooseMethodLabel">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
      </property>
      <property name="text">
       <string>Choose method</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="runPushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="chooseImagePushButton">
      <property name="text">
       <string>Choose Image</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
      </property>
      <property name="text">
       <string>PENNY CALCULATOR</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="numberOfPenniesLabel">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
